### ISSUE TITLE
https://prelude.dhall-lang.org returning 404

### ISSUE THREAD

**Issue by erjenkins29** [dhall-lang/dhall-lang#1240]
## Context

The Prelude README states:

```markdown
To browse the latest release of the Prelude package, you can visit:

https://prelude.dhall-lang.org/
```

The website and other documentation also makes references to this link.  Browsing to this URL however returns a 404 error.  On browsing there, I'm forwarded to https://store.dhall-lang.org/Prelude-v21.1.0, which is the url that ultimately raises the 404.

**Comment by SiriusStarr** [comment]
#1237 fixes this.

**Comment by SiriusStarr** [comment]
This has been fixed by #1237, so closing it out.  Apologies for the interruption!
